flink-core/src/main/java/org/apache/flink/configuration/QueryableStateOptions.java

```diff
@@ -109,7 +109,7 @@
 	/** Option whether the queryable state proxy and server should be enabled where possible and configurable.
 	 *
 	 * <p>Queryable state proxy and server are still more experimental features, hence disabled unless they are enable
-	 * in user configuration. */
+	 * in user configuration.  */
```

Comment:
Maybe we can remove the added blank character.

Reply:
> Maybe we can remove the added blank character.

Very careful code review :)
I will remove it.